### DISCUSSION TITLE
Does this app work for Devices that are learned using the Tuya IR blaster?

### DISCUSSION THREAD

**Discussion by alison22000** [milo526/homebridge-tuya-web#345] [Q&A]
I just wanted to ask if the Tuya IR blaster will work with this plugin. I havbe tried adding devices that are learned and used on the app, but it tells me there was an error searching for that device ID. I have also tried adding the IR blaster ID itself. It would be wonderful if these could work, but if not, I can get another Broadlink. I believe the IR blaster I have was branded for Lenovo, but it is a Tuya.

**Comment by milo526** [comment]
To the best of my knowledge, these IR blasters cannot be connected to homebridge at the moment but @bkarlson seems to have found a way to get that to work? So maybe they can help you out here!

**Reply by alison22000** [reply]
Is there anyway I can contact them? I’m not too familiar with GitHub yet. I’d love to know how the progress is going.

**Reply by bkarlson** [reply]
you can use the following plugin for devices controlled by an IR blaster, a/c and fans supported with all functions, as for other devices it's just generic on/off switch, which is sufficient in many usecases:

https://github.com/prasad-edlabadka/homebridge-tuya-ir

homebridge-tuya-web does not support IR-devices as Tuya does not include them in `/homeassistant` API. I have also tried Tuya LAN and Tuya homebridge -- those plugins cannot see devices behind bridges at all, so no use.

**Reply by bkarlson** [reply]
Another way (probably even better one) is to set up scenes in your Tuya app (Smart -> Tap-to-run), e.g. "Turn the a/c on", and such automation will be exposed as a Scene via this plugin (homebridge-tuya-web). It works really well for curtains (three scenes: open, closed, ajar), can work for simple on/off devices as well.
